prow/config.yaml

```diff
@@ -10608,6 +10643,74 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
+- cron: "0 0,12 * * *" # mutual blocking with ci-kubernetes-e2e-gci-gce-scalability-stable1
```

Comment:
Blocking with the other job? I can't see anywhere in the config that it's blocked on it.
If you mean that the cron is such that it ensures they don't overlap, I would not call it blocking.

Reply:
well, prow don't have blocking mechanism, and the two job timeout is much shorter than 6h. Or we can wait and then drop the cvm job once we have 1.9.

Reply:
I wasn't asking for making the jobs block on each other (that is not even the case currently and we ensure they don't overlap by just timing them in such a way). I meant that the comment in that line is giving a wrong meaning, as you use the term "mutual blocking", while it's not really blocking per se.